test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -156,11 +174,13 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
     }
 
     "redirects correctly when the call to the export api fails" in {
```

Comment:
Could this potentially be re-worded, at first glance `redirects correctly` doesn't tell you where the redirect should be to?

Reply:
These export API ones are ones that Sam added, so I'll ask him.